### ISSUE TITLE
Screen readers double-reading option text in ComboBox and Dropdown options

### ISSUE THREAD

**Issue by ConorWuertz** [OfficeDev/office-ui-fabric-react#7408]
### Environment Information 

- __Package version(s)__: 6.111.2,  repros on latest demo page version
- __Browser and OS versions__: Latest Chrome

### Please provide a reproduction of the bug in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/6.114.0/#/examples/dropdown
 
#### Actual behavior:
 
Using the NVDA screenreader to navigate the options in the dropdown or combo box results in the "text" provided to that option being read twice. This happens because when the component's aria label and title props are not defined, the component falls back to using the text prop as a default ariaLabel and title. This causes screen-readers to duplicate the text when reading it out. This behavior is unwanted and not necessary.
 
#### Expected behavior:
 
This text should be read only once. Aria label and title attributes should only be rendered if provided via props.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes, will submit shortly
 
Requested priority: Normal
 
Products/sites affected: Microsoft Teams

**Comment by MdAmeerMind** [comment]
Hi, Narrator still narrating option text twice for (Chrome + NVDA) (JAWS + IE) ,
It is reproducible on test release site provided above https://fabricweb.z5.web.core.windows.net/oufr/6.118.0/#/examples/ComboBox#Variants
